<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgLoginClass</class>
 <widget class="QDialog" name="QDlgLoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDlgLogin</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>551</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="MainWindow.qrc">:/images/images/preview.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelUsername">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditPassword"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnForgetPwd">
       <property name="text">
        <string>找回密码</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnRegister">
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBoxRemeber">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="labelTips">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxUsername">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
